nsv13/code/modules/coffee/machine/impressa.dm

```diff
@@ -0,0 +1,243 @@
+#define BEAN_CAPACITY 10
+/*
+ * impressa coffee maker
+ * its supposed to be a premium line product, so its cargo-only, the board cant be therefore researched
+ */
+
+/obj/machinery/coffeemaker/impressa
```

Comment:
I attempted to find out whether this is a real brand of coffeemaker. I'm still not sure but apparently it's also a line of incontinence support products.

Reply:
Could alter it to some other name

Reply:
It is in fact also a coffeemaker, so I would like it if we could rename this to avoid trademark infringement

Reply:
It has now been renamed to Pendulum
A random name I got from this random name generator!
https://www.fantasynamegenerators.com/scientific-device-names.php